crates/h264/src/sps.rs

```diff
@@ -205,15 +281,20 @@ impl Sps {
 }
 
 #[derive(Debug, Clone, PartialEq)]
-/// Sequence parameter set extension.
+/// The Sequence Parameter Set extension.
```

Comment:
The following fields are variable length integers we should specify that somehow.